src/app/ApplicationTemplate.Tests/Presentation/NavigationTestsBase.cs

```diff
@@ -14,6 +14,9 @@
 
 namespace ApplicationTemplate.Tests
 {
+	/// <summary>
+	/// Gives access to navigation methods to reproduce the standard app behavior.
```

Comment:
I'm not sure about "to reproduce the standard app behavior". You could do anything with those methods.